.circleci/install_triggers

```diff
@@ -1,5 +1,3 @@
 ^\.circleci/
 ^environment\.yml$
 ^pyproject.toml$
-^setup\.py$
-^setup\.cfg$
```

Comment:
`setup.cfg` still exists, did you remove that here on purpose?

Reply:
Yes, it now only contains the configuration for pycodestyle, so it does not affect the (development) installation of the tool anymore.

Reply:
Makes sense. Would it make sense to remove all unused element in `setup.cfg` here? It looks like it contains more than the pycodestyle config:

https://github.com/ESMValGroup/ESMValCore/blob/a3557ecdde7d1b9e857610779105fe75049f7e04/setup.cfg#L1-L38

Reply:
Looks like I messed up the last merge with `main`. Should be fixed now.

Reply:
And then I forgot to move the changes from #2456 to pyproject.toml. Should be really fixed now :facepalm: